### PR TITLE
Kevin

### PR DESCRIPTION
**Pull request by drmeister**

### CODE REVIEW

File: src/widget_selection.lisp [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+  (:metaclass traitlets:traitlet-class))
+
+(defun %label-to-value (k object)
+      (gethash 'k (slot-value object '_options_dict)))
+
+ (defun %labels-to-values (k obj)
+   (loop for v in k collect (%label-to-value v obj)))
+
+(defun %value-to-label (value obj)
+  (with-slots (options) obj
```
Comment (drmeister) [inline src/widget_selection.lisp]: This needs to do what the Python code does:
https://github.com/drmeister/spy-ipykernel/blob/master/ipywidgets/widgets/widget_selection.py#L16

(car (rassoc value (options obj) :test #'equal))

**Review by drmeister — COMMENTED** [review]
I can't see anything wrong -so I'm going to approve these changes

**Comment by drmeister** [comment]
Hmm, I can't approve the changes because I authored the pull request.  I'll ask Alex to review this pull request.

**Review by Bike — APPROVED** [review]
I'm only looking cursorily but it seems fine. Assuming it works on your machine. I'm kind of curious why slot names are changing, though.

**Pull request merged**